src/config.js

```diff
@@ -12,16 +12,66 @@ const avoidDefault = value => {
   }
 };
 
-const conf = convict({
+const password = value => {
+  const adminPass = owasp.test(value);
+  if (!adminPass.strong) {
+    adminPass.errors.forEach(error => console.error(error));
+    throw new Error('Admin pass is default or not strong enough.');
+  }
+};
+
+const remoteSTConfig = {
+  server: {
+    doc: 'SensorThings sandbox API server',
```

Comment:
nit: if we use something different to SensorUp, this might not be a sandbox anymore :)